packages/components/src/DataList/components/DataListLayoutInternal/DataListItems.tsx

```diff
@@ -27,11 +30,24 @@ export function DataListItems<T extends DataListObject>({
         return (
           <>
             {elementData.map((child, i) => {
+              const item = data[i];
+
               return (
-                <div className={styles.listItem} key={`${data[i].id}`}>
+                <div
+                  onMouseEnter={() => setHover(item.id)}
+                  onMouseLeave={() => setHover(undefined)}
+                  className={styles.listItem}
+                  key={`${item.id}`}
```

Comment:
Why is it in template strings?

Reply:
hmmm can't remember, i think it has something to do with key's needing to be a string.

Reply:
looks like it doesn't have to be in template strings haha